tests/HUBViewControllerTests.m

```diff
@@ -1849,6 +1849,7 @@ - (void)testRenderingUpdatesContentInsetBeforeAndAfterRendering
 
 - (void)testProposedContentInsetIsDefaultIfHeaderMissing
 {
+    #if !TARGET_OS_TV
```

Comment:
Should this encompass the whole test (including the method signature) as with `testAdaptingOverlayComponentCenterPointToKeyboard`?